pkg/cli/alpha/config-gen/README.md

```diff
@@ -7,7 +7,7 @@ Supports:
 - Generating CRDs and RBAC from code
 - Generating webhook certificates for development
 - Selectively enabling / disabling components such as prometheus and webhooks
-  - See [types.go](apis/v1alpha1/types.go) for a list of components
+  - See [types.go](types.go) for a list of components
```

Comment:
Regards its docs I think we need to move in the direction : https://github.com/kubernetes-sigs/kubebuilder/issues/2542

Reply:
Yeah, that sounds good.
I can make the docs once the implementation is good.
See someone is working on the thread, if there is a successful one give out ahead of this pr, I can try attach the docs based on that.
Otherwise we may have a follow up?
Would you think this work?